extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -72,12 +71,26 @@ export class TargetSelectionPage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
 			}).component();
 
+		this._certMigrationRequiredInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
```

Comment:
when is this visible?  does it show up for SQL VM and SQL DB selection?

Reply:
It's disabled by default via CSS.

Reply:
that's a great question @brian-harris . 

regardless of the default setting, the display will be controlled by like 296 in OnPageEnter where we set the display:
await utils.updateControlDisplay(this._certMigrationRequiredInfoBox, this.migrationStateModel.tdeMigrationConfig.shouldAdsMigrateCertificates());

Will shouldAdsMigrateCertificates() always return false for SqlDB or VM cases?  

also is there a possibility of tdeMigrationConfig being null for SqlDB or VM cases?

Reply:
@AkshayMata

`Will shouldAdsMigrateCertificates() always return false for SqlDB or VM cases?`
Yes because we call `this._resetTdeConfiguration();` whenever we select a target that's not MI.  This will by default return false.

`also is there a possibility of tdeMigrationConfig being null for SqlDB or VM cases?`
No, `tdeMigrationConfig` is instantiated in `MigrationStateModel` before we know what the target is.